.github/workflows/tests_linters.yaml

```diff
@@ -1,16 +1,16 @@
 name: Tests and Linters 🧪
 
-on: [ push, pull_request ]
+on: [ pull_request ]
```

Comment:
Does this make it run only one time (once the PR is open)?

Reply:
It does work every time it's modified, removed the push because I don't think we need the checks unless it's a PR